Ports/cpio/patches/0001-Use-global-program_name-variable-from-gnu-dir.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Liav A <liavalb@gmail.com>
+Date: Sat, 25 Jun 2022 05:45:29 +0300
+Subject: [PATCH] Use global program_name variable from gnu dir
```

Comment:
Is there a particular reason for doing this though?

Reply:
It won't compile without this.

Reply:
I added a brief description to ensure it is clear why we need this patch.